content-tool/tool/src/java/org/sakaiproject/content/tool/ListItem.java

```diff
@@ -4475,6 +4475,18 @@ public String getServiceName()
 		// This is used when asking if the styles of the service should be used.
 		return ServerConfigurationService.getString("ui.service", "Sakai");
 	}
-	
+
+	/**
+	 *Parse the id of the file returns the original file name WL-3168
```

Comment:
You shouldn't have references to the Jira issue in the code. As ideally we would want to take this patch as it and send back to Sakai and having a reference to local Oxford issues doesn't makes sense.